lib/index.d.ts

```diff
@@ -47,36 +47,33 @@ export declare type StepOptionComplete = {
 	message?: string;
 	submessage?: string;
 };
-export declare type StepConfigWelcome = {
-	type: typeof STEP_WELCOME;
-	options?: StepOptionWelcome;
+export declare type StepOptionsMap = {
+	welcome: StepOptionWelcome;
+	userConsent: never;
+	document: StepOptionDocument;
+	poa: StepOptionPoA;
+	face: StepOptionFace;
+	complete: StepOptionComplete;
 };
-export declare type StepConfigUserConsent = {
-	type: typeof STEP_USER_CONSENT;
-	options?: never;
-};
-export declare type StepConfigDocument = {
-	type: typeof STEP_DOCUMENT;
-	options?: StepOptionDocument;
-};
-export declare type StepConfigPoA = {
-	type: typeof STEP_POA;
-	options?: StepOptionPoA;
-};
-export declare type StepConfigFace = {
-	type: typeof STEP_FACE;
-	options?: StepOptionFace;
-};
-export declare type StepConfigComplete = {
-	type: typeof STEP_COMPLETE;
-	options?: StepOptionComplete;
+export declare type StepConfigMap = {
+	[Type in StepTypes]: {
+		type: Type;
+		options?: StepOptionsMap[Type];
+	};
 };
-export declare type StepConfig = StepConfigWelcome | StepConfigUserConsent | StepConfigDocument | StepConfigPoA | StepConfigFace | StepConfigComplete;
+export declare type StepConfigWelcome = StepConfigMap["welcome"];
+export declare type StepConfigUserConsent = StepConfigMap["userConsent"];
+export declare type StepConfigDocument = StepConfigMap["document"];
+export declare type StepConfigPoa = StepConfigMap["poa"];
+export declare type StepConfigFace = StepConfigMap["face"];
+export declare type StepConfigComplete = StepConfigMap["complete"];
+export declare type StepConfig = StepConfigWelcome | StepConfigUserConsent | StepConfigDocument | StepConfigPoa | StepConfigFace | StepConfigComplete;
 export declare type UICustomizationOptions = {
 	colorBackgroundSurfaceModal?: string;
 	colorBorderSurfaceModal?: string;
 	borderWidthSurfaceModal?: string;
 	borderStyleSurfaceModal?: string;
+	borderRadiusSurfaceModal?: string;
```

Comment:
Addressed in PR #1397

Reply:
Idk but for some reasons these lines were missed from `development` branch. This should fix it.